_i18n/en/documentation/general/statistics.md

```diff
@@ -0,0 +1,30 @@
+AntennaPod has a statistics screen that lets you view the number of listening hours per subscription and per month. However, these statistics aren’t always as precise as they could be.
+
+## Why the stats aren’t precise
+To track statistics, AntennaPod records two things for each episode:
+* **Played duration**: how long the episode was played (e.g. 6 minutes if a 3-minute episode was played twice)
+* **Last played**: the date and time when playback was last paused or stopped for the episode
+
+Both are updated each time playback is paused or stopped. This information is used to calculate totals by adding up the 'played duration' of all episodes whose 'last played' date falls within a given month.
+
+This approach means that if you listen to 7 minutes of an episode today and 38 minutes of that episode tomorrow, all 45 minutes are attributed to tomorrow. In our experience, this is not an uncommon scenario; many users listen to episodes across several days. As a result, a significant portion of listening time may be attributed to the wrong day, making day-by-day statistics unreliable.
+
+The same issue applies across months. If an episode is played both in June and July, all listening time is attributed to July. Similarly, if an episode is played both in December of one year and in January of the next, it is attributed to the second year.
+
+In the statistics screen, listening time may therefore be attributed to the wrong month or year. But because each period is much larger, it is less likely that a single episode is listened in two periods, and it’s thus less likely that statistics are wrong. We accept the (smaller) risk of inaccurate data at the month or year level, but we do not want to accept the (larger) risk at the day level.
```

Comment:
This sounds very negative, almost like a bug. I would phrase this more positively.

```suggestion
This approach means that if you listen to 7 minutes of an episode today and 38 minutes of that episode tomorrow, all 45 minutes are attributed to tomorrow. The same applies across months. If an episode is played both in June and July, all listening time is attributed to July. Similarly, if an episode is played both in December of one year and in January of the next, it is attributed to the second year.

The monthly statistics are usually accurate because they look at a longer time period. However, day-by-day statistics would have a higher risk of becoming unreliable, which is why AntennaPod does not offer those.
```